hack/update-bindata.sh

```diff
@@ -29,6 +29,8 @@ if [[ ! -d "${KUBE_ROOT}/examples" ]]; then
 	exit 1
 fi
 
+export PATH="${GOPATH}/bin:${PATH}"
```

Comment:
I was expecting all of the `go get github.com/jteeuwen/go-bindata/go-bindata` calls to be removed - we should never ever be pulling random stuff from head of any tree that we don't control.

That aside, this runs from GOPATH/bin, but the compile that we make locally is in _output/bin/ ?